
 ================= 2025-03-25 09:43:58,045 ============== 
 ERROR Dao.GenericDao`1 
 Error guardando objeto: Model.MAT_CAT_SEVERIDADSystem.Data.Entity.Infrastructure.DbUpdateException: An error occurred while updating the entries. See the inner exception for details. ---> System.Data.Entity.Core.UpdateException: An error occurred while updating the entries. See the inner exception for details. ---> System.Data.SqlClient.SqlException: Infracción de la restricción UNIQUE KEY 'IX_UNIQUE_NIVEL_MAT_CAT_SEVE'. No se puede insertar una clave duplicada en el objeto 'dbo.MAT_CAT_SEVERIDAD'. El valor de la clave duplicada es (5.0000).
Se terminó la instrucción.
   en System.Data.SqlClient.SqlConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   en System.Data.SqlClient.SqlInternalConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   en System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)
   en System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)
   en System.Data.SqlClient.SqlDataReader.TryConsumeMetaData()
   en System.Data.SqlClient.SqlDataReader.get_MetaData()
   en System.Data.SqlClient.SqlCommand.FinishExecuteReader(SqlDataReader ds, RunBehavior runBehavior, String resetOptionsString, Boolean isInternal, Boolean forDescribeParameterEncryption, Boolean shouldCacheForAlwaysEncrypted)
   en System.Data.SqlClient.SqlCommand.RunExecuteReaderTds(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, Boolean async, Int32 timeout, Task& task, Boolean asyncWrite, Boolean inRetry, SqlDataReader ds, Boolean describeParameterEncryptionRequest)
   en System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method, TaskCompletionSource`1 completion, Int32 timeout, Task& task, Boolean& usedCache, Boolean asyncWrite, Boolean inRetry)
   en System.Data.SqlClient.SqlCommand.RunExecuteReader(CommandBehavior cmdBehavior, RunBehavior runBehavior, Boolean returnStream, String method)
   en System.Data.SqlClient.SqlCommand.ExecuteReader(CommandBehavior behavior, String method)
   en System.Data.SqlClient.SqlCommand.ExecuteDbDataReader(CommandBehavior behavior)
   en System.Data.Common.DbCommand.ExecuteReader(CommandBehavior behavior)
   en System.Data.Entity.Infrastructure.Interception.DbCommandDispatcher.<Reader>b__c(DbCommand t, DbCommandInterceptionContext`1 c)
   en System.Data.Entity.Infrastructure.Interception.InternalDispatcher`1.Dispatch[TTarget,TInterceptionContext,TResult](TTarget target, Func`3 operation, TInterceptionContext interceptionContext, Action`3 executing, Action`3 executed)
   en System.Data.Entity.Infrastructure.Interception.DbCommandDispatcher.Reader(DbCommand command, DbCommandInterceptionContext interceptionContext)
   en System.Data.Entity.Internal.InterceptableDbCommand.ExecuteDbDataReader(CommandBehavior behavior)
   en System.Data.Common.DbCommand.ExecuteReader(CommandBehavior behavior)
   en System.Data.Entity.Core.Mapping.Update.Internal.DynamicUpdateCommand.Execute(Dictionary`2 identifierValues, List`1 generatedValues)
   en System.Data.Entity.Core.Mapping.Update.Internal.UpdateTranslator.Update()
   --- Fin del seguimiento de la pila de la excepción interna ---
   en System.Data.Entity.Core.Mapping.Update.Internal.UpdateTranslator.Update()
   en System.Data.Entity.Core.EntityClient.Internal.EntityAdapter.<Update>b__2(UpdateTranslator ut)
   en System.Data.Entity.Core.EntityClient.Internal.EntityAdapter.Update[T](T noChangesResult, Func`2 updateFunction)
   en System.Data.Entity.Core.EntityClient.Internal.EntityAdapter.Update()
   en System.Data.Entity.Core.Objects.ObjectContext.<SaveChangesToStore>b__35()
   en System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   en System.Data.Entity.Core.Objects.ObjectContext.SaveChangesToStore(SaveOptions options, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction)
   en System.Data.Entity.Core.Objects.ObjectContext.<>c__DisplayClass2a.<SaveChangesInternal>b__27()
   en System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   en System.Data.Entity.Core.Objects.ObjectContext.SaveChangesInternal(SaveOptions options, Boolean executeInExistingTransaction)
   en System.Data.Entity.Core.Objects.ObjectContext.SaveChanges(SaveOptions options)
   en System.Data.Entity.Internal.InternalContext.SaveChanges()
   --- Fin del seguimiento de la pila de la excepción interna ---
   en System.Data.Entity.Internal.InternalContext.SaveChanges()
   en System.Data.Entity.Internal.LazyInternalContext.SaveChanges()
   en System.Data.Entity.DbContext.SaveChanges()
   en Dao.GenericDao`1.Save(T entity) en C:\Antilavado SQL SERVER\fsv_mvc\Dao\GenericDao.cs:línea 333 
  

 ================= 2025-03-25 09:43:58,097 ============== 
 ERROR Blo.GenericBlo`1 
 System.Exception: No puede guardar niveles que ya existen.
   en Dao.GenericDao`1.Save(T entity) en C:\Antilavado SQL SERVER\fsv_mvc\Dao\GenericDao.cs:línea 369
   en Blo.GenericBlo`1.Save(T entity) en C:\Antilavado SQL SERVER\fsv_mvc\Blo\GenericBlo.cs:línea 241 
  

 ================= 2025-03-25 09:43:58,146 ============== 
 ERROR View.Controllers.BaseController 
 System.Exception: No puede guardar niveles que ya existen.
   en Blo.GenericBlo`1.Save(T entity) en C:\Antilavado SQL SERVER\fsv_mvc\Blo\GenericBlo.cs:línea 247
   en View.Controllers.Matriz.CatSeveridadController.Save(MAT_CAT_SEVERIDAD data) en C:\Antilavado SQL SERVER\fsv_mvc\View\Controllers\Matriz\CatSeveridadController.cs:línea 67 
  
